framework/src/play/mvc/Router.java

```diff
@@ -354,13 +349,9 @@ public static void routeOnlyStatic(Http.Request request) {
                 if (route.matches(request.method, request.path, request.format, request.domain) != null) {
                     break;
                 }
-            } catch (Throwable t) {
-                if (t instanceof RenderStatic) {
-                    throw (RenderStatic) t;
-                }
-                if (t instanceof NotFound) {
-                    throw (NotFound) t;
-                }
+            } catch (RenderStatic | NotFound e) {
+                throw e;
+            } catch (Throwable ignore) {
```

Comment:
Why did you put a silent catch here, I don't think it is a good idea

Reply:
@xael-fry This is exactly how the previous code worked. 
@cbxp But I agree, it would be better to add logging for those exceptions.

Reply:
@asolntsev You are right sorry, maybe check that adding a log will not generate too many logs, it is maybe a normal behavior.

Reply:
@xael-fry As this PR doesn't change the behaviour here, I suggest not to introduce more changes, but make them separately if needed. This change was suggested by the IDE and just makes it more obvious that other Throwables are just ignored.

Reply:
@xael-fry So, can we merge this PR?